test/support/utils.h

```diff
@@ -68,45 +71,58 @@ const_size(const T (&)[N]) noexcept
 #define EXPECT_EQ_RANGES(expected, actual, message)                                                                    \
     ::TestUtils::expect_equal(expected, actual, __FILE__, __LINE__, message)
 
+inline void
+exit_on_error()
```

Comment:
why do we need this new function?

Reply:
This function required for exit with the same error code, and called if error occurred in tests (in all tests) in my new tests from main, if we have errors during test.